flight/targets/colibri/fw/chconf.h

```diff
@@ -405,7 +405,7 @@
  *          @p panic_msg variable set to @p NULL.
  */
 #if !defined(CH_DBG_ENABLE_STACK_CHECK) || defined(__DOXYGEN__)
-#define CH_DBG_ENABLE_STACK_CHECK       TRUE
```

Comment:
just out of curiosity, why are we disabling this check? does the stack overflow alarm still work? i can imagine there is a small performance improvement with this, but how much and is it worth it?